src/TaskManager.java

```diff
@@ -0,0 +1,148 @@
+            System.out.println("Nothing to list");
+            Elvis.printHorizontalLines();
+        } else {                //Print out everything in the list
+            Elvis.printHorizontalLines();
+            System.out.println("Here are the tasks in your list: ");
+
+            for (int i = 0; i < listCount; i++) {
+                System.out.print(i + 1 + "." + "[" + list[i].getTaskType() + "]");
+                System.out.print("[" + list[i].getStatus() + "] " + list[i].getDescription());
+
+                // Check if the task is of type 'D' (Deadline) and cast it to Deadline if so
+                if (list[i] instanceof Deadline) {
+                    Deadline deadlineTask = (Deadline) list[i];
+                    System.out.println(" (by: " + deadlineTask.getDate() + ")");
+                } else if (list[i] instanceof Event) {
+                    Event eventTask = (Event) list[i];
+                    System.out.println(" (from: " + eventTask.getStartTime() + " to: " + eventTask.getEndTime() + ")");
+                } else {
+                    System.out.println("");
+                }
+            }
+            Elvis.printHorizontalLines();
+        }
+    }
+
+    public static void markTask(int numberInput) {
+        int nthTask = numberInput - 1;
+        if (numberInput > listCount || nthTask < 0 || listCount == 0) {
+            System.out.println("No such Task!");
+            return;
+        }
+        list[nthTask].setStatus(true);
+        Elvis.printHorizontalLines();
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println(nthTask + 1 + "." + "[" + list[nthTask].getTaskType() + "]" +
+                "[" + list[nthTask].getStatus() + "] " + list[nthTask].getDescription());
+        Elvis.printHorizontalLines();
+    }
+
+    public static void unmarkTask(int numberInput) {
+        int nthTask = numberInput - 1;  //Remember that index of list starts from 0
+        if (numberInput > listCount || nthTask < 0 || listCount == 0) {
+            System.out.println("No such Task!");
+            return;
+        }
+        list[nthTask].setStatus(false);
+        Elvis.printHorizontalLines();
+        System.out.println("OK, I've marked this task as not done yet:");
+        System.out.println(nthTask + 1 + "." + "[" + list[nthTask].getTaskType() + "]" +
+                "[" + list[nthTask].getStatus() + "] " + list[nthTask].getDescription());
+        Elvis.printHorizontalLines();
+    }
+
+    public static void insertToDo(String inputBuffer) {
+        Elvis.printHorizontalLines();
+        list[listCount] = new ToDo(inputBuffer.trim().replace("todo ", ""));
+        System.out.println("Got it. I've added this task:");
+        System.out.println("[" + 'T' + "]" +
+                "[" + list[listCount].getStatus() + "] " + list[listCount].getDescription());
+        System.out.println("Now you have " + (listCount+1) + " task(s) in the list.");
```

Comment:
I think it is suggested to have white spaces between the operators? I noticed this is some other lines too.
```suggestion
        System.out.println("Now you have " + (listCount + 1) + " task(s) in the list.");
```